src/main/java/com/doongji/nestalk/controller/v1/user/dto/FindEmailRequest.java

```diff
@@ -0,0 +1,21 @@
+package com.doongji.nestalk.controller.v1.user.dto;
+
+import io.swagger.annotations.ApiModelProperty;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+public class FindEmailRequest {
+
+    @ApiModelProperty (value = "사용자 이름", required = true)
```

Comment:
사소한거지만 전체적으로 코드 정리가 필요한 부분이 좀 있는거같아요.

```suggestion
    @ApiModelProperty(value = "사용자 이름", required = true)
```

Reply:
인텔리제이 자동정렬 체크가 잘못되어있었던거 같아요. 수정했습니다.